arrows/core/close_loops_exhaustive.cxx

```diff
@@ -152,10 +152,16 @@ ::stitch( vital::frame_id_t frame_number,
     all_matches[f] = pool.enqueue(match_func, f);
   }
 
+  std::map<vital::frame_id_t, track_pairs_t> all_matches_done;
```

Comment:
You might want to add a comment to document that this loop and transferring to a new map is for thread synchronization.  E.g. make sure all the parallel jobs are finished before merging the tracks.